app/views/quotes/_quote.html.erb

```diff
@@ -1,23 +1,35 @@
 <li class="py-4">
   <div class="flex space-x-3 text-sm">
-    <%= link_to quote.user, class: "self-center" do %>
-      <% gravatar_for quote.user, class: "h-8 w-8 rounded-full" %>
+    <% if quote.anonymous %>
+      <div class="self-center">
+        <%= image_tag 'anonymous.svg', class: "w-7 h-7 rounded-full" %>
```

Comment:
Waarom is deze w/h-7 en de normale gravatar 8?

Reply:
Joah zag er wat netter uit